charts/astronomer/values.yaml

```diff
@@ -37,10 +37,6 @@ images:
     repository: quay.io/astronomer/ap-cli-install
     tag: 0.13.1
     pullPolicy: IfNotPresent
-  e2eTest:
```

Comment:
the helm chart itself had e2etest setup in it?!

Reply:
what was using these values? does that need cleaned up too?

Reply:
I believe all the other parts were removed in https://github.com/astronomer/astronomer/pull/910. This is cleanup up pieces that one didn't get.